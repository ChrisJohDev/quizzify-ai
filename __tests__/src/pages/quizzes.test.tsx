import {render, screen }from '@testing-library/react';
import Quizzes from '@/pages/quizzes';
import '@testing-library/jest-dom'

describe('ID: QAI:base-TC014 - Quizzes Page', () => {
  it('renders correctly', () => {
    const { container } = render(<Quizzes />);
    expect(container).toMatchSnapshot();
  });
});