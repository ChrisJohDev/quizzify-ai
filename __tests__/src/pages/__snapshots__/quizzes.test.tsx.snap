// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quizzes - snapshot test renders correctly 1`] = `
<div>
  <div
    class="container mx-auto flex flex-col flex-fill quizPage"
  >
    <h1
      class="text-center"
    >
      Create your quiz
    </h1>
    <div
      class="col-md-6 offset-md-3 flex flex-col info"
    >
      <form
        action="/quizResponse"
        class="mx-auto"
        method="POST"
      >
        <label
          for="issue"
        >
          Number of questions:
        </label>
        <select
          id="issue"
          name="amount"
        >
          <option
            value="10"
          >
            10
          </option>
          <option
            value="15"
          >
            15
          </option>
          <option
            value="20"
          >
            20
          </option>
          <option
            value="25"
          >
            25
          </option>
        </select>
        <label
          for="input-element"
        >
          Subject:
        </label>
        <input
          id="input-element"
          name="subject"
          placeholder="Leave blank for general knowledge"
          type="text"
        />
        <br />
        <input
          type="submit"
          value="Submit"
        />
      </form>
    </div>
  </div>
</div>
`;
